### DISCUSSION TITLE
ControlNet with FP16 in NEXT.SD

### DISCUSSION THREAD

**Discussion by blacklig** [vladmandic/sdnext#2260] [Q&A]
Hi folks,

anyone knows how to get those models running in NEXT.SD?

https://huggingface.co/lllyasviel/control_v11p_sd15_inpaint/tree/main

what I like about them is they say they are already FP16 so I expect half the generation time, but cannot get it run. It just throws errors with config problems and I dont really know how to fix this.

Running "normal" control nets with "proper yaml config" files is no issue and that works to me no pronlem (like those https://huggingface.co/lllyasviel/ControlNet-v1-1/tree/main)

Any help would be very appreciated :)

**Comment by brunogcar** [comment]
rename either the safesensor or the yaml file as they must match

that should work, have not tried those files,  let  us know

if not ask at https://github.com/Mikubill/sd-webui-controlnet
